api/spec/spec_handler.go

```diff
@@ -452,6 +453,15 @@ func (d *specHandler) UpdateSpecFromOpts(opts map[string]string, spec *api.Volum
 			} else if v == "LoadBalancer" || v == "loadBalancer" {
 				spec.Sharedv4ServiceSpec.Type = api.Sharedv4ServiceSpec_LOADBALANCER
 			}
+		case api.SpecSharedv4ExternalAccess:
+			if spec.Sharedv4ServiceSpec == nil {
+				spec.Sharedv4ServiceSpec = &api.Sharedv4ServiceSpec{}
+			}
+			if v == "true" || v == "True" {
```

Comment:
It is a little more reliable to use `strconv.ParseBool(v)` (see line 494 below for an example - https://github.com/libopenstorage/openstorage/pull/1905/files#diff-2e2db81cab87c799e2f24955e8905fc6fdbb5a5fcdfc837dbe8ad4f00c4f4b39R494)